### ISSUE TITLE
Seperators for items

### ISSUE THREAD

**Issue by WinKhantAung** [guangzhouxia/JTSegmentControl#1]
I wanna add seperators between each item in JTSegmentControl. Could u suggest me how to make it?

**Comment by guangzhouxia** [comment]
add seperators in right of JTItemView ,set it's seperators  show when the item is not first and last...i think ....

**Comment by WinKhantAung** [comment]
yep...i got it guangzhouxia. These are my lines of code in screenshot.
![screen shot 2017-03-08 at 1 41 02 pm](https://cloud.githubusercontent.com/assets/16660802/23693905/545f9eca-0405-11e7-9dd7-273b87f794f3.png)
